### PR TITLE
SC: update contracts solidity version to 0.5.6

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
This PR applied solidity 0.5.6 to the contracts related with Service Chain.
To `go generate`, solidity 0.5.6 version is needed.

- openzeppelin is updated to 2.3.0.

How to switch solidity 0.5.6
```
$ brew unlink solidity  
$ brew install solidity@0.5.6
$ brew link solidity@0.5.6
$  solc --version
solc, the solidity compiler commandline interface
Version: 0.5.6+commit.b259423e.Darwin.appleclang
```
## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by KimKyungup** [comment]
@jeonghonah @nogang @aidan-kwon PTAL. :)

**Comment by jeonghonah** [comment]
Please check following directories.

```
 computationcost/
 cypress/
 feepayer/
 precompiledContracts/
 reward/
 validatesender/
```

**Comment by KimKyungup** [comment]
> Please check following directories.
> 
> ```
>  computationcost/
>  cypress/
>  feepayer/
>  precompiledContracts/
>  reward/
>  validatesender/
> ```

I think we don't have to change solidity version of `cypress, reward`. Because they are already included in cypress. :) But others can be changed. I will apply in next PR after talking with the person who added them.

**Comment by KimKyungup** [comment]
Dear, reviews. :)
I think you don't need to review openzeppline. 
I just copied them from https://github.com/OpenZeppelin/openzeppelin-contracts/tree/v2.3.0.
I possible, you can check the diff with https://github.com/OpenZeppelin/openzeppelin-contracts/tree/v2.3.0.

File: contracts/externals/openzeppelin-solidity/contracts/examples/SampleCrowdsale.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
```
Comment (ehnuje) [inline contracts/externals/openzeppelin-solidity/contracts/examples/SampleCrowdsale.sol]: Including this line, some lines are updated to `0.5.0`, not to `0.5.6`. Is it intended?

Comment (KimKyungup) [reply contracts/externals/openzeppelin-solidity/contracts/examples/SampleCrowdsale.sol]: Yes. All openzeppelin codes are used 0.5.0 not specific 0.5.x.
But in our code (bridge, token), I used 0.5.6 version.

**Pull request merged**